requirements.txt

```diff
@@ -16,7 +16,8 @@ netifaces==0.10.7
 networkx==2.1.0
 psutil==5.4.5
 pycryptodome==3.6.1
-raiden-libs==0.1.2
+# raiden-libs==0.1.2
```

Comment:
Do not comment this out, simply replace it. Commenting it causes pip to fail. It has happened before: https://github.com/raiden-network/raiden/pull/2023